src/lib.rs

```diff
@@ -85,7 +85,7 @@ impl zed::Extension for ScalaExtension {
             CompletionKind::Keyword => "",
             _ => return None,
         };
-        let name = completion.label;
+        let name = completion.label.replace("  ", " ").replace("\n", "");
```

Comment:
I don't think labels currently should have newlines, do you have an example where it happens?

Reply:
The `toList` example above is one

Reply:
Could you post a snippet? I can't see to reproduce the same behaviour.

https://github.com/user-attachments/assets/744bb159-7a3c-41cd-87f3-0b052e528d8c

Reply:
Try `toList` on a tuple

Reply:
That works, it's a bug in Metals. We should not show that in the label.

Reply:
Could we merge this for now, because it wouldn't make things worse if there are no newlines and would improve the situation for now

Reply:
Aure, but I wonder about the nix related stuff. Is that intended?

Reply:
I've added it as an addition. If you don't want it, I can remove it